Extractor/RebrowseTriggerManager.cs

```diff
@@ -220,16 +218,15 @@ CancellationToken token
                 try
                 {
                     var values = item.DequeueValues();
-                    var valueTime = values[0]
-                        .GetValue<DateTime>(default)
-                        .ToUnixTimeMilliseconds();
-                    var id = item.ResolvedNodeId.ToString();
+                    var valueTime = values[0].GetValue<DateTime>(default).ToUnixTimeMilliseconds();
```

Comment:
I am pretty sure this will fail. Try this: `DateTime.MinValue.ToUnixTimeMilliseconds()`. I think that will throw an exception.

Reply:
Sorry, I wasn't being clear. `default(DateTime) == DateTime.MinValue`, but `DateTime.MinValue.ToUnixTimeMilliseconds()` will fail, so `default(DateTime).ToUnixTimeMilliseconds()` will also fail.